templates/networkpolicy.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.networkPolicy.enabled }}
```

Comment:
We need an Apache License v2 header.

Reply:
Hi @lewismc - I've added the license header ;)